src/runtime/operator.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Python.Runtime
+{
+    internal static class OperatorMethod
+    {
+        public static Dictionary<string, Tuple<string, int>> OpMethodMap { get; private set; }
+
+        private static Dictionary<string, string> _pyOpNames;
```

Comment:
Add a comment, describing contents of this Dictionary, or name it so that it would become clear.

Reply:
Maybe something like
> Maps the compiled method name in .NET CIL (e.g. `op_Addition`) to the Python operator (e.g. `__add__`).

Reply:
I think this dictionary can be deleted since it is never used.